float32 goal_error
float32 goal_linear_error
float32 goal_angular_error
int32 option
int32 xyz
int32 sign
---
float32[] error_result
float32 time_take
geometry_msgs/Pose final_pose
---
float32[] error_way
geometry_msgs/Pose[] way_pose